.travis.yml

```diff
@@ -3,8 +3,9 @@ os: linux
 sudo: required
 
 go:
-  - 1.x
-go_import_path: k8s.io/minikube
+  - 1.12.x
```

Comment:
If possible, leave this at 1.x so that we will continue to auto-update when v1.13 comes along.

Reply:
Done.